packages/host/src/server.ts

```diff
@@ -358,13 +358,30 @@ const outputHeuristics: Heuristic[] = [
 			message: /^Quitting: /,
 		},
 		action: function(parsed) {
-			this.emit("_quitting");
-
 			// On windows the Factorio server will wait for input on the console input buffer
 			// when an error occurs.  Since we can't send input there we terminate the server
 			// process when this happens.
 			if (process.platform === "win32" && parsed.message === "Quitting: multiplayer error.") {
 				this._server!.kill();
+				process.nextTick(() => {
```

Comment:
To delay the "killing Factorio" message after the last line is logged so that it doesn't look like it was killed while still working. I will add a comment for this.